Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/packages.config

```diff
@@ -2,30 +2,51 @@
 <packages>
   <package id="Autofac" version="4.4.0" targetFramework="monoandroid71" />
   <package id="Autofac.Extras.MvvmCross" version="4.0.2" targetFramework="monoandroid71" />
+  <package id="Microsoft.CSharp" version="4.3.0" targetFramework="monoandroid71" />
```

Comment:
Probably a dependency from the other packages.. the new .NET standard approach relies heavily on packages instead of local versions of .NET framework.